### ISSUE TITLE
Accessibility Bug: There is no visual representation ‘please wait’ for the busy indicator as highlighted in red.

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3111]
Description: There is no visual representation ‘please wait’ for the busy indicator as highlighted in red.

Expected: There is no visual representation "Please wait" for the busy indicator as highlighted in red 

Attached the screenshot
![image](https://user-images.githubusercontent.com/32538291/91549532-b57ae700-e944-11ea-8489-7e92ac8db56c.png)

**Comment by salarenko** [comment]
Hey @puru-hk could you be more specific what is the problem?
The description of the issue is not very helpful,

**Comment by puru-hk** [comment]
As per the accessibility standard. if there is a images for busy indicator it should have alternative text for example "Please wait". Please see the attachment. This is fundamental core component "Busy indicator". 
Please let me know if you need more details

**Comment by olshansky** [comment]
@puru-hk Are you about the example with text below the busy indicator bubbles? Like [the first example here ](https://sapui5.hana.ondemand.com/#/entity/sap.m.BusyIndicator/sample/sap.m.sample.BusyIndicator)

As for the other cases, I can't find any requirements for additional visual representation. 
Busy indicator should have alternative text for screen readers, and our component has it. 
I believe the issue could be closed, or please add a reference to the requirements or/and examples.

**Comment by Sanjay-1312** [comment]
Hello Colleagues,

While loading information, visually toast message should be provided such as 'Please Wait'.
So please fix the issue.

Thanks and Regards
Sanjay

**Comment by puru-hk** [comment]
@Sanjay-1312 If you could attached the screen shot it would be good.

**Comment by Lokanathan-k** [comment]
Please do not test this is not fixed yet in the Fundamental NGX as this change is addressed in the styles under version 18 and the current version of style is 17 and this will be changed once the style version is migrated to the 18 or the latest version of style
